cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -595,6 +597,10 @@ def _get_ssh_client(self, raise_on_error=True, timeout=5000):
                 .read(["host_key"])[0]
                 .get("host_key")
             )
+            if not host_key and not self.skip_host_key:
```

Comment:
What will be in the command log if we raise an exception here?
If we check the host key, there will be an exception raised in [611](https://github.com/cetmix/cetmix-tower/blob/b248286480a536e246b2c10b95670cb99340be59/cetmix_tower_server/models/cx_tower_server.py#L611).